performanceTest.php

```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```

Comment:
@martin-schilling Can you remove these headers please. These headers can be configured for each project.